### PR TITLE
refactored Scaffolding and added MsIdentityScaffolding

### PR DESCRIPTION
**Pull request by deepchoudhery**
- On [commit ](https://github.com/dotnet/MSIdentityScaffolding/commit/bb495975ad2f10937558a9b2bce1d9093644e73d)from dotnet/MsIdentity.
- added .github for filing different issues, questions, and feature requests.
- changed Scaffolding.sln --> All.sln.
      - dotnet-aspnet-codegenerator, and the dotnet-scaffolding projects can be loaded with Scaffolding.slnf
      - dotnet-msidentity can be loaded using MsIdentity.slnf
- added scripts/ folder to build and install dotnet-scaffolding, dotnet-aspnet-codegenerator, and dotnet-msidentity separetely.
- new folder structure
      image

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/eng/                                   @aspnet/build
```
Comment (vijayrkn) [inline .github/CODEOWNERS:4]: We should change these code ownerships. It should be us.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Contributing
 ======
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md) in the Home repo.
+Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md) in the AspNetCore repo.
```
Comment (vijayrkn) [inline .github/CONTRIBUTING.md:4]: Scaffolding

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue with ASP.NET Core
```
Comment (vijayrkn) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Should fix the contacts.

**Review by vijayrkn — CHANGES_REQUESTED** [review]
Since there are no other code changes, approving.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (deepchoudhery) [reply .github/CONTRIBUTING.md:4]: The link is pointing to the AspNetCore repo so we should name it such. I can switch it to dotnet/AspNetCore for more clarity.

File: .github/CODEOWNERS (hunk 1 above)
Comment (deepchoudhery) [reply .github/CODEOWNERS:4]: This file was not intended to stay. We're not making any changes in eng/ other than in Versions.props and Versions.Details.xml. These changes are minor and we don't really need to be requesting for reviews there.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (deepchoudhery) [reply .github/ISSUE_TEMPLATE/config.yml:3]: > Should fix the contacts.

These are in addition to the bug, feature and question links pertaining scaffolding and msidentity scaffolding. Take a look at https://github.com/dotnet/efcore/issues/new/choose for reference. We can just remove them but no harm in giving devs the options to report those issues.

**Review by vijayrkn — APPROVED** [review]

**Pull request merged**